gateway/crates/config/src/lib.rs

```diff
@@ -186,6 +186,15 @@ pub struct SubgraphConfig {
     pub entity_caching: Option<EntityCachingConfig>,
     /// Subgraph specific message signatures config
     pub message_signatures: Option<MessageSignaturesConfig>,
+    #[cfg(feature = "dev")]
+    /// The path of an SDL schema file for the subgraph.
+    pub file: Option<PathBuf>,
```

Comment:
I find `file` to be too ambiguous, it should be `schema_path` IMO as we already use `_path` suffix and `schema` to explicit about what it is

Reply:
Fair enough will change it

Reply:
Done